content/tutorials/getting-started/index.md

```diff
@@ -166,60 +175,113 @@ Now, you can directly skip to the [generate a package]({{< ref "#generate-packag
+Enter the name of your package, then hit **Enter**
 
-Therefore, you need to enter `computerdatabase.BasicSimulation` as the class name.
+```
+Choose the load generators location
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Location AP - Hong kong
+[2] Location AP - Tokyo
+[3] Location AP Pacific - Mumbai
+[4] Location AP SouthEast - Sydney
+[5] Location Europe - Dublin
+[6] Location Europe - Paris
+[7] Location SA East - São Paulo
+[8] Location US East - N. Virginia
+[9] Location US West - N. California
+[10] Location US West - Oregon
+```
 
-Then, click on the **Next** button.
+Here we configure the Simulation itself.
+Choose the location where you want to spawn your load generators and generate traffic from.  
+For demonstration purposes we will choose the unarguable best city : `6`
 
-Next, you'll need to select your package name. It is important to select the package that contains the simulation you'd like to run. Then, click on the **Next** button.
+```
+Enter the number of load generators
+```
 
-In the final step, you can choose where you want the injection to take place, and click on the **Save** button.
+Define how many instances of load generators will be spawned in this region. Care, each load generator instance will consume credits while it runs. 
 
-Your simulation will send the traffic from the location you've selected.
+```
+Start simulation using simulation class name: computerdatabase.ComputerDatabaseSimulation
+Created simulation named gettingStartedSimulation with ID '<simulation-id>'
 
-The three next steps are optional and you can learn more about them [in the dedicated section]({{< ref "../../reference/user/simulations/" >}}), but for now, let's just simply click **Save**.
+Specify --simulation-id <simulation-id> if you want to start a simulation on Gatling Enterprise,
+or --simulation 90f60c5c-bee8-4a19-9655-866d4c49eb1d if you want to create a new simulation on Gatling Enterprise.
+See https://gatling.io/docs/gatling/reference/current/core/configuration/#cli-options/ for more information.
 
-{{< alert tip >}}
-If you would like to write your own script as a first test, you can modify the BasicSimulation file in the space as shown in the video. If you would like to learn more about creating different types of tests and writing your own scripts check out the [Gatling Academy](https://gatling.io/academy/). Please bear in mind you get 60 credits for free but if your test needs more, it will stop before it should finish.
-{{< /alert >}}
+Simulation successfully started; once running, reports will be available at https://cloud.gatling.io/o/<organization-id>/simulations/reports/<report-id>
+```
 
-That's it, you have created your first simulation on Gatling Enterprise!
+Click on the provided link to see the live reporting of your simulation run. 
 
 ### Using maven, gradle or sbt {#create-simulation-buildtools}
```

Comment:
I think we should merge this part with the bundle part, and keep the advanced explanation from the bundle